requirements.txt

```diff
@@ -1,7 +1,16 @@
-beautifulsoup4
-aiohttp
-async_timeout>=3.0,<4.0
-urllib3
-requests
-lxml
-tqdm
+aiohttp==3.7.4.post0
+async-timeout==3.0.1
+attrs==20.3.0
+beautifulsoup4==4.9.3
+certifi==2020.12.5
+chardet==4.0.0
+click==7.1.2
```

Comment:
I am adding `click`, which was previously not declared. I presume it has always worked for you guys since you already have them pre-installed in your machines.

The issue will hit someone like me, who comes in with a clean environment with nothing pre-installed.

Reply:
This should be [taken care of here](https://github.com/sec-edgar/sec-edgar/blob/a488f676f435fcc4d5edad33e1b95a09fe6b01a5/setup.py#L78). To get the CLI, you have to `pip install secedgar[cli]`.

Reply:
Got it. Change reverted.